pshtt/pshtt.py

```diff
@@ -623,7 +654,7 @@ def https_check(endpoint):
         public_trust = True
         custom_trust = True
         public_not_trusted_string = ""
-        validation_results = cert_plugin_result.path_validation_result_list
+        validation_results = cert_plugin_result.certificate_deployments[0].path_validation_results
```

Comment:
As a user, I'd prefer a tool be inclusive rather than presume there's only ever one cert – but without doing a run of our namespace I'm similarly unsure whether this would impact folks we scan.

Reply:
@mcdonnnj I can just loop over all the results just to be safe. Given most times there will be only one certificate, the loop will get executed only once, resulting in a similar code flow like this one. What do you think?

Reply:
@mcdonnnj in case of multiple leaf certificates, do we consider trusted if at least one has validations_results successful? Or do we consider trust if all leaf certificates have validation_results successful?

Reply:
@SaptakS I'm waiting for a team consensus on which way we want to lean on multiple certificates. I would hesitantly say that if we're checking all leaf certificates, then they all must be successful to be trusted, but that's my stance.

Reply:
I have currently implemented it using that logic in the latest commit. So do check that out, if that works.

Reply:
I agree with what @mcdonnnj said.  All leaf certificates must be successful to be trusted.